### ISSUE TITLE
Can we get an ability to call a raw query

### ISSUE THREAD

**Issue by chugh97** [essuraj/elasticsearch-toolbox#13]
At the moment the editor uses the indexes found in the selected drop down to create the POST/GET query. What I would like is somewhere to call the war query directly. As we use elastic aliases we cant call them from this tool. If you can provide ability to call the raw queries that would be greatly appreciated.

**Comment by MystyxMac** [comment]
Yeah, we also use aliases and like to edit the raw url or at least put the aliases in the dropdown.
